src/api/spec/controllers/staging/staged_requests_controller_spec.rb

```diff
@@ -99,6 +99,65 @@
       it { expect(staging_project.staged_requests).to include(bs_request) }
       it { assert_select 'status[code=ok]' }
     end
+
+    context 'with an excluded request', vcr: true do
+      context 'when not providing the remove exclusion parameter' do
+        before do
+          create(:request_exclusion,
+                 bs_request: bs_request,
+                 number: bs_request.number,
+                 staging_workflow: staging_workflow)
+          login user
+          post :create,
+               params: { staging_workflow_project: staging_workflow.project.name,
+                         staging_project_name: staging_project.name, format: :xml },
+               body: "<requests><number>#{bs_request.number}</number></requests>"
+        end
+
+        it { expect(response).to have_http_status(:bad_request) }
+        it { expect(response.body).to match(/invalid_request/) }
+        it { expect(response.body).to match(/Error while removing excluded requests: Request #{bs_request.number} not found in Staging for project #{staging_workflow.project.name}/) }
+      end
+
+      context 'when providing the remove exclusion parameter' do
+        before do
+          create(:request_exclusion,
+                 bs_request: bs_request,
+                 number: bs_request.number,
+                 staging_workflow: staging_workflow)
+          login user
+          post :create,
+               params: { staging_workflow_project: staging_workflow.project.name,
+                         staging_project_name: staging_project.name,
+                         remove_exclusion: true,
+                         format: :xml },
+               body: "<requests><number>#{bs_request.number}</number></requests>"
+        end
+
+        it { expect(response).to have_http_status(:success) }
+        it { expect(staging_project.packages.pluck(:name)).to match_array([target_package.name]) }
+        it { expect(staging_project.staged_requests).to include(bs_request) }
+        it { assert_select 'status[code=ok]' }
+      end
+    end
+
+    context 'when providing the remove exclusion parameter' do
+      context 'and not having any excluded request' do
+        before do
+          login user
+          post :create,
+               params: { staging_workflow_project: staging_workflow.project.name,
+                         staging_project_name: staging_project.name,
+                         remove_exclusion: true,
```

Comment:
The same as before, `true`or `1`?

Reply:
@hennevogel and I discussed this yesterday and agreed on not caring what the content of the variable is as long as its "truthy", to be consistent with the rest of the API.

Reply:
however as @saraycp say, consistence is the key. Everywhere else, AFAICR, we use 1. It is expected to have 1 here too.

Reply:
I'll use a `'1'` in the specs and documentation, but just check for parameter presence in the controller